wallet/src/de/schildbach/wallet/ui/send/SendCoinsViewModel.kt

```diff
@@ -124,6 +124,7 @@ class SendCoinsViewModel @Inject constructor(
             }
             .launchIn(viewModelScope)
 
+        // TODO: the coin selector will need to use CoinJoinCoinSelector if CoinJoin is ON
         walletDataProvider.observeBalance(coinSelector = MaxOutputAmountCoinSelector())
             .distinctUntilChanged()
             .onEach(_maxOutputAmount::postValue)
```

Comment:
In the UI story for Mixing Information, this will need to be fixed where we select the correct CoinSelector based on the user selected `CoinJoinMode`.